src/sonic_ax_impl/mibs/vendor/cisco/ciscoPfcExtMIB.py

```diff
@@ -207,7 +207,7 @@ def requestsPerPriority(self, sub_id):
         else:
             return self._get_counter(port_oid, counter_name)
 
-    def indicationsPerPriority(self, sub_id):
+    def indications_per_priority(self, sub_id):
```

Comment:
>indications_per_priority [](start = 8, length = 24)

Suggest refactor this class also following other similar SubtreeMIBEntry. The updater should create OID dict and list. All query interfaces should be quite simple.